tests/platform_tests/cli/test_show_platform.py

```diff
@@ -163,7 +163,10 @@ def test_show_platform_syseeprom(duthosts, enum_rand_one_per_hwsku_hostname, dut
             pytest_assert(utility_cmd_output["stdout"].find(field) >= 0, "Expected field '{}' was not found on '{}'".format(field, duthost.hostname))
 
         for line in utility_cmd_output["stdout_lines"]:
-            pytest_assert(line in syseeprom_output, "Line '{}' was not found in output on '{}'".format(line, duthost.hostname))
+            if not line.startswith('-'):  # do not validate line '-------------------- ---- --- -----'
+                line_regexp = re.sub(r'\s+', '\s+', line)
```

Comment:
What is the purpose of this?

Reply:
In test we comparing outputs from python script:
`        utility_cmd = "sudo python -c \"import imp; \
            m = imp.load_source('eeprom', '/usr/share/sonic/device/{}/plugins/eeprom.py'); \
            t = m.board('board', '', '', ''); e = t.read_eeprom(); t.decode_eeprom(e)\"".format(duthost.facts["platform"])`

and from command "show platform syseeprom"

Output from python script has different number of spaces. Then when we do "pytest_assert(line in syseeprom_output," .... test fail because lines not identical. My fix replace spaces by \s+ and doing matching by regexp.